content/en/tracing/guide/setting_up_apm_with_kubernetes_service.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Setting up APM with Kubernetes Service
+kind: guide
+further_reading:
+- link: "/containers/kubernetes/apm/"
+  tag: "Documentation"
+  text: "Set up trace collection"
+- link: "/containers/cluster_agent/admission_controller"
+  tag: "Documentation"
+  text: "Admission Controller"
+---
+
+## Overview
+
+There are 3 main ways the Datadog Tracers send their data to the Datadog Agent for the proper handling. Those are: Host IP, Unix Domain Socket (UDS), and the Kubernetes service. These options are all setup in a way to ensure the APM data the application pod sends arrive to the Datadog Agent pod on the same node. This strategy is meant to properly balance the traffic and ensure the correct tagging of your data.
+
+Datadog recommends to utilize the UDS strategy to send data. However, the Kubernetes service can be used as an alternative option when the `hostPath` volumes required for UDS and the `hostPort` ports required for the Host IP method are not available.
+
+## Service setup
+
+In Kubernetes you can take advantage of [services][1] to help route traffic to downstream pods. One limitation of the default service configuration is that this routes the traffic to the downstream pods at random. In Kubernetes 1.22 the [Internal Traffic Policy feature][2] was added, giving the option to set the configuration `internalTrafficPolicy: Local` on the service. When set this directs the traffic from an application pod to the service's downstream pod *on the same node*. Maintaining that strategy of routing the APM data to the Datadog Agent on the same node.
+
+This service is created for you automatically in our Datadog Helm Chart and Datadog Operator on Kubernetes clusters with version 1.22.0 or above. You additionally need to enable the APM port option for your Agent with the below configuration.
+
```

Comment:
I'm wondering how necessary all of this information is for the user. If you think it is very necessary to explain this background, I'd recommend putting it in a section further down on the page. For me, the only vital information here is "The service is created for you if you used the Datadog Helm chart or Operator to deploy the Agent on a Kubernetes cluster v1.22.0+. Configuration is below."

Reply:
I trimmed it down quite a bit to:

> In Kubernetes 1.22 the [Internal Traffic Policy feature](https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/) was added, giving the option to set the configuration internalTrafficPolicy: Local on the service. When set this directs the traffic from an application pod to the service's downstream pod on the same node.
> 
> A service for the Agent using this feature is created for you automatically in our Datadog Helm Chart and Datadog Operator on Kubernetes clusters with version 1.22.0 or above. You additionally need to enable the APM port option for your Agent with the below configuration.